File: service/SurfWeb/Core/Dto/NewRecords/PopularMapDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Core.Dto.NewRecords
{
    public class PopularMapDto
    {
        public required string Id { get; set; }
        /// <summary>
        /// 名称
        /// </summary>
        public required string Name { get; set; }
        /// <summary>
        /// 难度
        /// </summary>
        public required string Difficulty { get; set; }
        /// <summary>
        /// 地图图片
        /// </summary>
        public required string Img { get; set; }
        /// <summary>
        /// 完成人数
        /// </summary>
        public int SurcessNumber { get; set; }
    }
}

File: service/Java/SurfWeb/SurfWeb/system/src/main/java/com/surfweb/system/dto/newrecords/PopularMapDto.java
package com.surfweb.system.dto.newrecords;

import lombok.Data;

@Data
public class PopularMapDto {

    private long id;

    /**
     * 名称
     */
    private String name;

    /**
     * 地图图片
     */
    private String img;

    /**
     * 难度
     */
    private String difficulty;

    /**
     * 完成人数
     */
    private int surcessNumber;

}
